assets/css/layouts.css

```diff
@@ -34,6 +34,8 @@ div.tiles > * {
 /* - flexbox shortcut - */
 .grid { display: grid; }
 .left { float: left; }
+.right { float: right; }
+.scroll { overflow: scroll; }
```

Comment:
this is out of scope for this PR. in the future, we may want to do `.scroll-x` and `.scroll-y` as helpers, but not yet.

Reply:
added helpers